README.md

```diff
@@ -23,54 +23,31 @@ iPSL is part of the [iTesla Tool](https://github.com/itesla/ipst), and thus, it
 If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use.
-Please consult our [Publication List](http://openipsl.readthedocs.io/en/latest/publications.html), for browsing the reference publications.
+Please consult our publication list, located within the Users' Guide package, in the Publications page, for browsing the reference publications.
 
 Preferably, please cite the following paper:
   - Maxime Baudette, Marcelo Castro, Tin Rabuzin, Jan Lavenius, Tetiana Bogodorova, Luigi Vanfretti, OpenIPSL: Open-Instance Power System Library — Update 1.5 to “iTesla Power Systems Library (iPSL): A Modelica library for phasor time-domain simulations”, SoftwareX, Volume 7, January–June 2018, Pages 34-36, ISSN 2352-7110, https://doi.org/10.1016/j.softx.2018.01.002
     - Link to Publisher's Website: [SoftwareX](https://www.sciencedirect.com/science/article/pii/S2352711018300050)
 
-You are also welcome to submit your contributions as stated above.
-
-## Copyright and License Information:
+You are also welcome to submit your contributions as stated below.
 
-The OpenIPSL is released under the same license as former iPSL project.
-All changes applied in this fork have the following copyright statement.
+## Contributing:
 
-### OpenIPSL:
-Copyright July 2017 - current [Luigi Vanfretti](https://github.com/lvanfretti), [Rensselaer Polytechnic Institute](http://rpi.edu), [ALSETLab](https://github.com/ALSETLab), Troy, NY.
+Contributions to the library are welcome and can be submitted in the form of pull requests to this repository. Please consult the [contribution guidelines](.github/CONTRIBUTING.md) for more information on how to contribute to the development of this library. Information about our current and past contributors can be found when the library is loaded into a Modelica-compliant software. The information is located within the Users' Guide package, in the Contact page.
 
-Copyright April 2016 - May 2017 [SmarTS Lab](http://openipsl.readthedocs.io/en/latest/community/community-home.html); Prof. [Luigi Vanfretti's](https://github.com/lvanfretti) research group formerly at KTH Royal Institute of Technology, Stockholm, Sweden, now [ALSETLab](https://github.com/ALSETLab) at Rensselaer Polytechnic Institute, Troy, NY, USA.
+If you want to submit your contributions to the OpenIPSL, note that we have a [Contributor's License Agreement (CLA)](.github/legal/README.md). Individual contributors should read, accept and virtually sign the [Individual Contributor License Agreement](.github/legal/cla-individual.md) for contributions to be included in the OpenIPSL. Contributions submitted by entities need to have the [Entity Contributor License Agreement](.github/legal/cla-individual.md) read, accepted and virtually signed. If you have any questions, please don't hesitate to <a href="mailto:luigi.vanfretti@gmail.com?subject=Contributing_to_OpenIPSL">contact us</a>.
```

Comment:
I would simply link to the CLA guide you have created which also explains the automatic procedure and simply remove the rest of the paragraph.